constellations/orion/src/domains/user_afh.go

```diff
@@ -0,0 +1,28 @@
+package domains
+
+import "errors"
+
+var TABLE_USERAFH = "user_afh"
+
+type UserAfh struct {
+	Id     uint `json:"id"`
+	UserId uint `json:"user_id"`
+	AfhId  uint `json:"afh_id"`
```

Comment:
same here. Make `afhId` for json, but `afh_id` for db

Json dictates what the field will look like when coming in and out of the controller.
DB dictates what the field looks like in the database.

Reply:
but the struct field names UserId and AfhId are fine.